src/borg/locking.py

```diff
@@ -141,7 +141,7 @@ def acquire(self, timeout=None, sleep=None):
             timer = TimeoutTimer(timeout, sleep).start()
             while True:
                 try:
-                    os.rename(temp_path, self.path)
+                    os.replace(temp_path, self.path)
```

Comment:
Not related to this current PR, but we should probably look at all uses of `os.rename` and change them to `os.replace` if possible.

Reply:
Right. There is some os.rename still in borg.cache and also in the testsuite. I'll have a look at that later.